common/headtracker/types/config.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+import "time"
+
+type HtrkConfig interface {
```

Comment:
Is this just a `Config`? given the `headtracker/types` context?
```suggestion
type Config interface {
```

Reply:
Thanks for pointing that out. That makes more sense